include/http2.hrl

```diff
@@ -210,7 +209,8 @@
           next_available_stream_id = 2 :: stream_id(),
           streams = [] :: [{stream_id(), stream_state()}],
           continuation_stream_id = undefined :: stream_id() | undefined,
-          content_handler = application:get_env(chatterbox, content_handler, chatterbox_static_content_handler) :: module()
+          content_handler = application:get_env(chatterbox, content_handler, chatterbox_static_content_handler) :: module(),
```

Comment:
record init, I think. I'm not sure but the only reason this is in the diff is because there's a comma at the end now. @tsloughter added this originally, and ironically, if I used commas the way he likes, this wouldn't have shown up in the PR